### PR TITLE
Add Fusion Builder component

### PR DESCRIPTION
**Pull request by strategio**
Also allow to pass an array of hooks objects to load for the integration.

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-5375

### CODE REVIEW

File: src/st/class-wpml-page-builders-app.php [hunk 1]
```diff
@@ -43,7 +43,15 @@ public function load_integration() {
 		if ( $factories ) {
 			foreach ( $factories as $factory ) {
 				$integration = $factory->create();
-				$integration->add_hooks();
+
+				if ( is_array( $integration ) ) {
+					foreach ( $integration as $hooks ) {
+						$hooks->add_hooks();
+					}
```
Comment (andreasciamanna) [inline src/st/class-wpml-page-builders-app.php]: @strategio I think it's better if:
- You move line 55 belove line 52
- Add `$this->integrations[] = $hooks;` below line 49

File: src/st/class-wpml-page-builders-app.php [hunk 2]
```diff
@@ -43,7 +43,15 @@ public function load_integration() {
 		if ( $factories ) {
 			foreach ( $factories as $factory ) {
 				$integration = $factory->create();
-				$integration->add_hooks();
+
+				if ( is_array( $integration ) ) {
+					foreach ( $integration as $hooks ) {
```
Comment (andreasciamanna) [inline src/st/class-wpml-page-builders-app.php]: @strategio can you give `$hook` a better name? Maybe `$integration_item` or something close to what it is supposed to be.

**Comment by andreasciamanna** [comment]
I don't know if this PR is still needed after what has been discussed in wpmlcore-5375

**Comment by strategio** [comment]
Closing as we decided to handle it in Core directly.

**Pull request closed without merging**